Analytics/Sources/Delivery/Analytics+Onboarding.swift

```diff
@@ -98,6 +98,38 @@ public extension Analytics {
     func trackOnboardingStart() {
         service.track(event: .did, element: .app, name: "onboarding_start")
     }
+    
+    func trackBenefitsStep() {
```

Comment:
Aren't these events already tracked by func trackOnboarding(_ step: OnboardingStep)?

Reply:
Ah yes you are right. I didn't see them in PostHog because the individual steps are set as properties on the `view_screen_onboarding` event. But it looks like we can make custom Actions in PostHog that let us treat these as unique events so we can do things like see how long users spend on each screen.

I will revert these parts of the PR.